adapter/src/dbt/adapters/fal_experimental/utils.py

```diff
@@ -109,21 +109,32 @@ def load_environments(base_dir: str) -> Dict[str, "BaseEnvironment"]:
 def create_environment(name: str, kind: str, config: Dict[str, Any]):
     from isolate.backends.virtualenv import VirtualPythonEnvironment
     from isolate.backends.conda import CondaEnvironment
+    from isolate.backends.remote import IsolateServer
 
     REGISTERED_ENVIRONMENTS: Dict[str, BaseEnvironment] = {
         "conda": CondaEnvironment,
         "venv": VirtualPythonEnvironment,
+        "remote": IsolateServer
     }
+
     env_type = REGISTERED_ENVIRONMENTS.get(kind)
+
     if env_type is None:
         raise ValueError(
             f"Invalid environment type (of {kind}) for {name}. Please choose from: "
             + ", ".join(REGISTERED_ENVIRONMENTS.keys())
         )
 
     parsed_config = {
-        'requirements': config.get('requirements', [])
-    }
+            'requirements': config.get('requirements', [])
+        }
+
+    if kind == "remote":
```

Comment:
how about adding a host to the profiles.yml file? we should enable remote mode through that as well

```
jaffle_shop:
  target: fal
  outputs:
    dev:
      type: duckdb
      path: "/Users/gorkemyurtseven/dev/duck_db_dbt_dump"
    fal:
      type: fal
      db_profile: dev
      host: http://146.83.102
```

Reply:
This was an open discussion point. We can either support per-environment level remotes OR we can support whole remote and whole local experience (by whether `host` is present in the credentials/config or not).

Reply:
I think it's easier to work with and think about if we do by project instead of by environment.